proto_helper/src/lib.rs

```diff
@@ -0,0 +1,163 @@
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::{Path, PathBuf};
+
+pub struct ProtoBuilder {
+    protoc_search_path: Option<PathBuf>,
+    files: Vec<PathBuf>,
+    import_paths: Vec<PathBuf>,
+    generate_grpc: bool,
+    transformations: Vec<Box<Fn(String) -> String>>,
+    grpc_transformations: Vec<Box<Fn(String) -> String>>,
+}
+
+impl ProtoBuilder {
+    // The path can be relative to the build script.
+    pub fn new() -> Self {
+        Self {
+            protoc_search_path: None,
+            files: vec![],
+            import_paths: vec![],
+            generate_grpc: false,
+            transformations: vec![],
```

Comment:
shall we add `replace_clippy` to `transformations` and `grpc_transformations` here, so we don't have to call `replace_clippy` explicitly later below?

Reply:
Sure, but then we can't do the `Default` thing, can we? I think it's clearer this way anyway.

Reply:
I implemented Default myself

Reply:
:+1: